### ISSUE TITLE
Still trying to relink files 

### ISSUE THREAD

**Issue by monteros53** [latenitefilms/BRAWToolbox#183]
Thanks for all the help with 
1. Giving sandbox access
2. instructions and steps for relinking files

I'm still getting this error message 
"Your xml was imported with the following warnings. Depending on the warnings you may want to modify the xml then import it again. The effect ID is invalid. (ref ="r4:/fcpxml[1'/media[1]/multicam[1]/mc-angle[1]/sync-clip[1]video[1]filter-viddeo[1]/@ref)

The other error I get when trying to drag the library, or project or event to relink is "No Braw toolbox clips were detected in the dragged FCPXML. Please ensure that you're dragging in a Library/Event/Project which contains BRAW toolbox clips"

**Comment by latenitefilms** [comment]
The **effect ID is invalid** most likely means you don't actually have the BRAW Toolbox effect installed?

See: 

https://brawtoolbox.io/installation/
https://brawtoolbox.io/troubleshooting/#the-effect-id-is-invalid

**Comment by monteros53** [comment]
HI, I have uninstalled and reinstalled Braw Toolbox.  Also uninstalled Final Cut and reinstalled. I've tried "Select Braw File" and get the message "Failed to get IBlackmagicRawJob in populateBRAWParametersWithReset."  When I drop the whole Library into Braw Toolbox to relink (great instructions), I get a list of files, copy the pathname to the file (have already granted Sandbox access to all my drives earlier) but never get the sandbox prompt and it never turns green.
